scripts/google/covid_mobility/TestCovidMobility.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from covidmobility import covid_mobility, csv_row_to_obj
+
+
+# NOTE: More testing will be added soon.
+
+
+class TestStringMethods(unittest.TestCase):
+    def test1_output(self):
+        # Get the output file.
+        covid_mobility('./test1.csv')
+        # Get the content of the expected output.
+        actual_f = open('./output/covid_mobility_output.mcf', 'r')
+        actual = actual_f.read().replace('\n', '')
+        actual_f.close()
+        # Get the content of the written output.
+        expected_f = open('./test1expectedoutput.mcf', 'r')
+        expected = expected_f.read().replace('\n', '')
+        expected_f.close()
+        self.assertEqual(actual, expected)
+
+    def test_csv_row_to_dict(self):
+        row = ["AE", "United Arab Emirates", "", "", "2/26/20", "-2", "1", "-3", "-2", "3", "1"]
+        covidmobility = covid_mobility('./test1.csv')
```

Comment:
What is this statement for?

Reply:
was testing to make sure that the CSV was converted to an Object correctly. but now i am using reader.